test/controller.test.ts

```diff
@@ -176,35 +176,6 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
       )
     })
 
-    it('should revert opening multiple vaults in the same operate call', async () => {
-      const actionArgs = [
```

Comment:
why remove this test?

Reply:
This should still be true right?

Reply:
yeah, removed by mistake, will add that.

Reply:
Added back.